include/mbgl/platform/time.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <chrono>
+
+namespace mbgl {
+namespace platform {
+
+// Returns the current time. Abstracted because some platforms
+// will not allow direct access to the current time via syscall.
+std::chrono::time_point<std::chrono::system_clock, std::chrono::seconds> now();
```

Comment:
why not `Timestamp` alias?

Reply:
That would cause a cyclic dependency I was trying to avoid. util/chrono.hpp depends on this and this would depend on util/chrono.hpp for this type alias.

Reply:
just declare the alias here too